src/TortoiseProc/LogDlg.cpp

```diff
@@ -431,10 +455,35 @@ BOOL CLogDlg::OnInitDialog()
 	reg.Replace(_T(':'),_T('_'));
 	m_History.Load(reg, _T("ref"));
 
+	m_FromDateHistory.SetMaxHistoryItems((LONG)CRegDWORD(_T("Software\\TortoiseGit\\MaxFromDateHistoryItems"), 5));
+	reg.Format(_T("Software\\TortoiseGit\\History\\LogDlg_FromDates\\%s"), g_Git.m_CurrentDir);
+	reg.Replace(_T(':'), _T('_'));
+	m_FromDateHistory.Load(reg, _T("FromDate"));
+
 	ShowStartRef();
 	return FALSE;
 }
 
+void CLogDlg::OnDestroy()
+{
+	__super::OnDestroy();
+
+	m_Brush.DeleteObject();

```

Comment:
I will drop this, because CBrush do DeleteObject() in its destructor.
See C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\atlmfc\include\afxwin1.inl Line 75